src/averageAges.js

```diff
@@ -37,7 +43,15 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(woman => woman.sex === 'f');
+  const isMother = withChildren !== undefined
```

Comment:
Consider renaming the variable 'isMother' to 'filteredWomen' for better readability. Variable names should start with a lowercase letter.